js/flightlog_parser.js

```diff
@@ -286,6 +286,7 @@ var FlightLogParser = function(logData) {
             features:null,                          // Activated features (e.g. MOTORSTOP etc)
             Craft_name:null,                        // Craft Name
             motorOutput:[null,null],                // Minimum and maximum outputs to motor's
+            collectiveRange:[-1000,1000],           // Minimum and maximum collective outputs (raw)
```

Comment:
1000 is the integer presentation of 1.0, which is equivalent to 12deg collective. It's not maximum. Maybe 1250 or 1500 would be a better maximum.

Reply:
I extended to 1500 and it still look acceptable
![collective_seekbar_v2](https://github.com/rotorflight/rotorflight-blackbox/assets/1812055/a98cbf78-a339-4f03-918b-854ef0f7462d)
Also added option to pass this information from firmware via header
`H collectiveRange:-1500,1500`

Reply:
Sorry about being indecisive. 1500 is probably too much as it is ~18deg.
Let's use 1250 ~= 15deg. Anything above that can be shown as "full collective"

Reply:
sure, done :)